velox/exec/FilterProject.cpp

```diff
@@ -117,6 +134,13 @@ RowVectorPtr FilterProject::getOutput() {
   auto* rows = localRows.get();
   rows->setAll();
   EvalCtx evalCtx(operatorCtx_->execCtx(), exprs_.get(), input_.get());
+
+  // Pre-load lazy vectors which are referenced by both expressions and identity
```

Comment:
If someone is using expression evaluation in a standalone manner, they would need to do this as well. Should we maybe document that standalone evaluation is not supported for Lazy Vectors?

Reply:
Good idea! How about we recommend both the pro option (figure out which lazy vectors are referenced within expressions and outside and load them, see FilterProject operator as reference) and the easy/cautious option (load all lazy vectors before running expr eval)

Reply:
Sounds great.

Reply:
done